exercises/robot-simulator/src/test/java/RobotTest.java

```diff
@@ -15,11 +15,11 @@ public void testRobotIsCreatedWithCorrectInitialPosition() {
 
```

Comment:
The first test should check both the position and direction, and the name should be updated accordingly 🙂